apply_extra.sh

```diff
@@ -1,6 +1,11 @@
 #!/usr/bin/sh
-ar x chrome.deb data.tar.xz
+set -e
+
+bsdtar -Oxf chrome.deb 'data.tar*' |
+  bsdtar -xf - \
+    --strip-components=4 \
+    --exclude='./opt/google/chrome/nacl*' \
+    ./opt/google/chrome
 rm chrome.deb
-tar -xf data.tar.xz --strip-components=4 ./opt/google/chrome
-rm data.tar.xz nacl*
-cp /app/bin/stub_sandbox chrome-sandbox
+
+install -Dm755 /app/bin/stub_sandbox chrome-sandbox
```

Comment:
super minor style nit to line up with the other code:

```suggestion
install -Dm 755 /app/bin/stub_sandbox chrome-sandbox
```